seed/ts-sdk/exhaustive/multiple-exports/src/api/resources/endpoints/client/Client.ts

```diff
@@ -0,0 +1,76 @@
+// This file was auto-generated by Fern from our API Definition.
+
+import type { BaseClientOptions } from "../../../../BaseClient.js";
+import { normalizeClientOptions } from "../../../../BaseClient.js";
+import { ContainerClient } from "../resources/container/client/Client.js";
+import { ContentTypeClient } from "../resources/contentType/client/Client.js";
+import { EnumClient } from "../resources/enum/client/Client.js";
+import { HttpMethodsClient } from "../resources/httpMethods/client/Client.js";
+import { Object_Client } from "../resources/object/client/Client.js";
+import { ParamsClient } from "../resources/params/client/Client.js";
+import { PrimitiveClient } from "../resources/primitive/client/Client.js";
+import { PutClient } from "../resources/put/client/Client.js";
+import { UnionClient } from "../resources/union/client/Client.js";
+import { UrlsClient } from "../resources/urls/client/Client.js";
+
+export declare namespace EndpointsClient {
+    export interface Options extends BaseClientOptions {}
+}
+
+export class EndpointsClient {
+    protected readonly _options: EndpointsClient.Options;
+    protected _container: ContainerClient | undefined;
+    protected _contentType: ContentTypeClient | undefined;
+    protected _enum: EnumClient | undefined;
+    protected _httpMethods: HttpMethodsClient | undefined;
+    protected _object: Object_Client | undefined;
+    protected _params: ParamsClient | undefined;
+    protected _primitive: PrimitiveClient | undefined;
+    protected _put: PutClient | undefined;
+    protected _union: UnionClient | undefined;
+    protected _urls: UrlsClient | undefined;
+
+    constructor(_options: EndpointsClient.Options) {
+        this._options = normalizeClientOptions(_options, "@fern/exhaustive", "0.0.1");
```

Comment:
`"@fern/exhaustive", "0.0.1"` is the same everywhere. Put it directly in normalizeClientOptions as literals

Reply:
Make the constructor parameter `options` and keep the private field as `_options`